internal/test/e2e/metrics_test.go

```diff
@@ -79,13 +79,6 @@ func TestMetrics(t *testing.T) {
 			return errors.New("no contract set churn metrics")
 		}
 
-		// check wallet metrics
-		wm, err := b.WalletMetrics(context.Background(), start, 10, time.Minute, api.WalletMetricsQueryOpts{})
```

Comment:
I'd keep this and add a skip just above it to signal that there is a test that is skipping some code and requires an update.

Reply:
wasn't aware you can `t.Skip` inside a test like that 👍